sources/packages/web/src/services/http/UserApi.ts

```diff
@@ -9,13 +9,33 @@ export default interface User {
 }
 
 export class UserApi extends HttpBaseClient {
-  public async checkUser(): Promise<User> {
+  public async checkUser(): Promise<boolean> {
+    let userExists = false;
     try {
       const response = await this.apiClient.get(
         "users/check-user",
         this.addAuthHeader(),
       );
-      return response.data as User;
+      if (response.status === 200) {
```

Comment:
We can add the same or similar npm that is used on API for HTTP Status Codes (e.g. HttpStatus.FORBIDDEN).

Reply:
There was a case which i am not able to reproduce now, where i got a user with id as blank. So this i think is still valid. Can discuss if needed.